traitcuration/static/js/uikit/uikit-icons.js

```diff
@@ -0,0 +1,167 @@
+/*! UIkit 3.5.3 | https://www.getuikit.com | (c) 2014 - 2020 YOOtheme | MIT License */
```

Comment:
I have mixed feelings about CDNs to be honest, mainly about security issues, although I could look at setting up referrer policies. On the other hand, I'm pretty sure using Yarn with Django will still require vendoring them, although I can search for a pip package to possibly solve that.

Reply:
https://github.com/epineda/django-yarn

I found that, I will try it out and see how well it works

Reply:
Actually it turns out Django does take care of security issues as far as I understand. So I would say since we are not going to use a lot of static dependencies, we can use CDNs to remove the overhead of static files

Reply:
> I have mixed feelings about CDNs to be honest, mainly about security issues

Yeah, that's true, I realise CDNs have their pros and cons. If we were developing something handling highly sensitive data, then maybe I wouldn't go for them. But public CDNs are supposed to be reasonably safe, and I feel this is the case where simplicity and speed of development are more important.

So, TL;DR, I support use of CDNs.